tools/xchain-suite/src/components/operations/GetBalance.tsx

```
@@ -5,7 +5,7 @@ import { ResultPanel } from '../ui/ResultPanel'
 import { CodePreview } from '../ui/CodePreview'
 import { generateGetBalanceCode } from '../../lib/codeExamples'
 import type { XChainClient, Balance } from '@xchainjs/xchain-client'
-import { assetToString, baseToAsset, formatAssetAmountCurrency, baseAmount } from '@xchainjs/xchain-util'
+import { assetToString, baseToAsset, formatAssetAmountCurrency, AssetType, type TokenAsset } from '@xchainjs/xchain-util'
```

ESLint

[error] Replace `·assetToString,·baseToAsset,·formatAssetAmountCurrency,·AssetType,·type·TokenAsset·` with `⏎··assetToString,⏎··baseToAsset,⏎··formatAssetAmountCurrency,⏎··AssetType,⏎··type·TokenAsset,⏎`

(prettier/prettier)